docs/pages/box.js

```diff
@@ -459,6 +456,16 @@ function Example() {
             {(props) => <Box width={60} height={60} rounding="circle" color={props.color} />}
           </CombinationNew>
         </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Elevation"
```

Comment:
new section around Elevation (will also house the new shadow options once those have been added)

Reply:
Could we add a background to simulate dark mode treatment? 


![image](https://user-images.githubusercontent.com/10593890/161814394-862c3f4c-69f5-49
![Screen Shot 2022-04-05 at 1 24 13 PM](https://user-images.githubusercontent.com/10593890/161814511-306d15c9-d865-4853-a02f-41a728c2d26d.png)
fc-bf71-71792473e001.png)

Reply:
I'm actually going to update this in the other PR when I add the border options to this section